tasks/devspace-runtime.yml

```diff
@@ -26,5 +29,11 @@
     - "nginxjenkins.devspace_nginxjenkins_1.state.running"
     - "redis.devspace_redis_1.state.running"
     - "seleniumhub.devspace_seleniumhub_1.state.running"
-    - "seleniumfirefox.devspace_seleniumfirefox_1.state.running"
-    - "seleniumchrome.devspace_seleniumchrome_1.state.running"
+    - "seleniumfirefox.devspace_seleniumfirefox_{{item}}.state.running"
+    - "seleniumchrome.devspace_seleniumchrome_{{item}}.state.running"
+  with_items:
```

Comment:
You might want to try http://docs.ansible.com/ansible/latest/playbooks_loops.html#looping-over-integer-sequences (and perhaps the `5` here could be a variable)